src/components/button/button.tsx

```diff
@@ -118,16 +126,16 @@ export const Button: FC<ButtonProps> = ({
   )
 }
 
-export const ButtonGroup: FC<ButtonGroupProps> = ({ children, alignment, ...rest }) => {
+export const DeprecatedButtonGroup: FC<ButtonGroupProps> = ({ children, alignment, ...rest }) => {
```

Comment:
**todo:** can you please add `/** @deprecated */` JS Doc to this component so that we get strike-through visual behaviour in our IDEs 🙏 

You may also need to add the same JS Doc to the `button/index.ts` file...not 100% sure though.